src/ru/skypro/Author.java

```diff
@@ -0,0 +1,44 @@
+package ru.skypro;
+
+import java.util.Objects;
+
+public class Author {
+    private final String authorsName;
```

Comment:
```suggestion
    private final String authorName;
```
поскольку апостроф нельзя использовать в наименованиях, то фраза вида `author's name` принято записывать как `authorName`